torch/_inductor/lowering.py

```diff
@@ -2113,8 +2111,12 @@ def is_aligned_backward(x):
     sdpa_constraint,
     warn=False,
 )
-make_fallback(torch.ops.aten._flash_attention_forward.default)
-make_fallback(torch.ops.aten._flash_attention_backward.default)
+make_fallback(
```

Comment:
Yup this was the inductor change I was going to point you too

Reply:
Seems like it still doesn't like it :(